dotCMS/src/main/java/com/dotmarketing/quartz/job/DistReindexJournalCleanupThread2.java

```diff
@@ -55,7 +56,6 @@ public void run() {
 			} catch (DotHibernateException e) {
 				Logger.error(this, e.getMessage(), e);
 			}
-    		DbConnectionFactory.closeConnection();

```

Comment:
Undo

Reply:
Also let's remove the `finally` block in the `execute` method, since it is not needed